packages/api-client/src/odsql/clauses.ts

```diff
@@ -1,11 +1,36 @@
 import { field, string } from '.';
 
+const formatFields = (fields: string[]) => {
+    if (!fields || fields.length === 0) return '';
+    return `${fields.map(field).join(', ')},`;
+};
+
+/**
+ * Perform full-text search on all fields. It matches the text fields that contain strings beginning with the searched string.
+ * @param text_fields Fields to search in
+ * @param text Text to search for
+ * @returns An ODSQL query, ready to be used as a `where` clause
+ */
+export const textSearch = (text: string, text_fields: string[] = []) =>
```

Comment:
Is there a reason to use snake case instead of camel case ?

Reply:
Absolutly no reason. I read too much python these days.